nautobot/core/models/generics.py

```diff
@@ -33,7 +31,7 @@ class PrimaryModel(BaseModel, ChangeLoggedModel, CustomFieldModel, RelationshipM
     tangible or logical resources on the network, or within the organization.
     """
 
-    tags = TaggableManager(through=TaggedItem)
+    tags = TaggableManagerField(through=TaggedItem)
```

Comment:
Another potential impact to plugins that we need to evaluate: is it possible for a plugin to remain compatible with versions of Nautobot both before and after this change? Since after this change any plugin using PrimaryModel as a base class will need to generate a migration for its models, but that migration won't be applicable to older versions of Nautobot... :-/

Reply:
As mentioned above the migrations will need to be generated, but only for consistency with the model definition in the source. No schema changes will be required outside of that.

Reply:
OK, but these migrations won't run under Nautobot 1.2.x because they'll be referencing a class that doesn't yet exist in 1.2.x, right?